### ISSUE TITLE
residual network example on  cifar10 not converging

### ISSUE THREAD

**Issue by psnegi** [tflearn/tflearn#58]
Ran residual_network_cifar10.py in example/image folder.
 At epoch 1   val_acc: 0.3040, after that is oscillates between 0.09xx and 0.10xx.
 
Another example from the same folder,  network_in_network.py gives accuracy on validation set upto .81.

**Comment by aymericdamien** [comment]
What GPU are you using? Is it an old one (CUDA compute compatibility 3.0)? I had similar issues with a GTX770 and ReLU activations, but on any GTX 9xx or Tesla K40c, haven't see any issue.
Otherwise, can you try to lower gradient clipping?

**Comment by psnegi** [comment]
I am using Tesla K40c. Using gradient clipping of .1  increased the val-acc to .67 but again it jumped  to .09 and .10 range.

**Comment by aymericdamien** [comment]
What is your CUDA version? And are you using CuDNN?
I tried it on a Tesla k40c (on CentOS 7), but I didn't get any issue and my model is converging.
Maybe be sure that you have the latest driver version too. I think this error is related to computation in CUDA or TensorFlow.

**Comment by psnegi** [comment]
After updating to  cuda 7.5, cuda dnn v4 and tensorflow .8 it is converging  with val_acc in the range [.83 - .84]. Thank you for the suggestions.

**Comment by aymericdamien** [comment]
Ok, I am adding some image preprocessing and augmentation (), this should raise validation performances. Also I think there might be an issue with learning rate not decreasing. I plan to investigate that and make an implementation closer to ResNet paper.
